src/components/CreditCard.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+
+function CreditCard(props) {
+    let {type, number, expirationMonth, expirationYear, bank, owner, bgColor, color} = props;
+  return (
+    <div>
+        <p>{props.type}</p>
+        <p>{props.number}</p>
```

Comment:
Here you should render multiple "***" followed by the last 4 numbers of the card